doc/api/esm.md

```diff
@@ -95,8 +95,26 @@ When loaded via `import` these modules will provide a single `default` export
 representing the value of `module.exports` at the time they finished evaluating.
 
 ```js
-import fs from 'fs';
-fs.readFile('./foo.txt', (err, body) => {
+// foo.js
+module.exports = { one: 1 };
+
+// bar.js
+import foo from './foo.js';
+foo.one === 1; // true
```

Comment:
Ignorable nit: as this is a file, not a REPL, maybe add `assert()` or `console.log()` here?